testData/solutions/win32.json5

```diff
@@ -6509,6 +6509,23 @@
             }
         },
         "configure": [
+            // Remove current system wallpaper files.
+            {
+                "type": "gpii.windows.rm",
+                "path": "${{environment}.APPDATA}\\Microsoft\\Windows\\Themes\\CachedFiles",
+                "options": {
+                    "recursive": true,
+                    "silent": true
+                }
+            },
+            {
+                "type": "gpii.windows.rm",
+                "path": "${{environment}.APPDATA}\\Microsoft\\Windows\\Themes\\TranscodedWallpaper",
+                "options": {
+                    "recursive": false,
```

Comment:
Maybe you can remove this option since the default value for "recursive" option is "false" as seen here: https://github.com/GPII/windows/pull/261/files#diff-864b1fb848cfe2a0afe44120b1955365R1929

Reply:
Yes, doing it.